LongWarOfTheChosen/Localization/XComGame.rus

```diff
@@ -82,6 +82,12 @@ PsiShortNames[7]="Пс."
 PsiShortNames[8]="Маг."
 
 [Spark X2SoldierClassTemplate]
+; LWOTC Translated
++AbilityTreeTitles[0]="Битва Будущего"
++AbilityTreeTitles[1]="Непробиваемость"
++AbilityTreeTitles[2]="Машина войны"
++AbilityTreeTitles[3]="XCOM"
+; End Translated
```

Comment:
Excellent, this is where it should go. Please remove the duplicate from *Localization/LW_PerkPack_Integrated/LW_PerkPack_Integrated.rus*

Reply:
done